src/routes/queues.ts

```diff
@@ -88,11 +91,11 @@ const getDataForQueues = async (
       const status =
         query[name] === 'latest' ? statuses : (query[name] as JobStatus[])
       const jobs = await queue.getJobs(status, 0, 10)
-
+      const formatter = formatJob(queue.options?.dataFormatter)
```

Comment:
I think that it is better to incapsulate the formatting part inside the adapters (inside `getJobs`), therefore, making `AdapterOptions` as `private` field.

Reply:
Yasss, maybe this is more correct. Will update 👍

Reply:
I tried to do this but reached the extent of my typescript knowledge and am bumbling around trying to figure it out. Sorry no more time to spend on it now.